micrometer-test/build.gradle

```diff
@@ -32,7 +32,7 @@ dependencies {
     api 'com.github.tomakehurst:wiremock-jre8-standalone'
     java11TestImplementation 'com.github.tomakehurst:wiremock-jre8-standalone'
 
-    api 'org.mockito:mockito-core'
+    api libs.mockitoCore4
```

Comment:
We should make sure to call out this change from Mockito 5 to 4 in the release notes.

Reply:
I added the `release-notes` label to the issue.